build/args/all.gn

```diff
@@ -47,4 +47,4 @@ enable_cet_shadow_stack = false
 is_cfi = false
 
 # TODO: fix this once sysroots have been updated.
-use_qt = false
+use_qt = true
```

Comment:
I think we can just delete this from our args; the default is true.

Reply:
0b5486a6e2d789c1a07c54dd3448be12fd882f13